web3/pm.py

```diff
@@ -132,7 +135,7 @@ def _get_package_name(self, package_id: bytes) -> str:
         pass
 
     @abstractmethod
-    def _get_all_package_ids(self) -> Tuple[bytes]:
+    def _get_all_package_ids(self) -> Iterable[bytes]:
```

Comment:
I'm confused by these being `Iterable[...]` rather than `Tuple[...]`.  I'm pretty sure they should be `Tuple` types.  Did `mypy` yell at you?

Reply:
>  Did mypy yell at you?

Yup, b/c in the [implementation](https://github.com/ethereum/web3.py/pull/1507/files#diff-b76a8371a5c2d30f739ddcd21e7a6b95R249) we use the `@to_tuple` decorator, which `mypy` recognizes as an `Iterable` b/c of the `yield` pattern rather than a `Tuple`